lib/components/src/controls/Boolean.tsx

```diff
@@ -70,8 +70,8 @@ const Label = styled.label(({ theme }) => ({
     },
   },
 
-  'input:checked ~ span:last-of-type, input:not(:checked) ~ span:first-of-type': {
-    background: theme.background.bar,
+  'input:checked ~ span:first-of-type, input:not(:checked) ~ span:last-of-type': {
+    background: theme.background.app,
     boxShadow: `${opacify(0.1, theme.appBorderColor)} 0 0 2px`,
     color: theme.color.defaultText,
     padding: '7px 15px',
```

Comment:
Let's chat with design about this change -- what is the expected text/background style of these toggle switches @MichaelArestad?

I'm leaning towards this implementation:
```  'input:checked ~ span:first-of-type, input:not(:checked) ~ span:last-of-type': {
    background: theme.background.bar,
    boxShadow: `${opacify(0.1, theme.appBorderColor)} 0 0 2px`,
    color: theme.color.inverseText,
    padding: '7px 15px',
```

when tested with the default custom theme, it looks like this:
![image](https://user-images.githubusercontent.com/17681528/124973291-f4f87200-dff0-11eb-992d-09206f57f40a.png)

Reply:
> what is the expected text/background style of these toggle switches

Good question. @domyen would know better than me. I'm not super familiar with Storybook theming yet.

What I might expect is that the text and background styles of the unselected boolean control is:
* The active item's background color could match the theme's background color .
* The active item's text color should be higher contrast against it's background than the inactive item's color/background contrast.

For example:
<img width="416" alt="image" src="https://user-images.githubusercontent.com/1123119/125129098-29853000-e0bc-11eb-86ae-b26b2defe9fa.png">

Reply:
Thanks @MichaelArestad -- @Nazeeh21 implemented this beautifully 🚀